### PR TITLE
Update kubernetes version and support mount propagation

### LINKED ISSUE #185: Support mount propagation.
We should support mount propagation after CRI supports it https://github.com/kubernetes/kubernetes/pull/46444.

### PR DESCRIPTION
**Pull request by miaoyq**
fixes #185

/cc @Random-Liu 

Signed-off-by: Yanqiang Miao <miao.yanqiang@zte.com.cn>

### CODE REVIEW

File: pkg/server/container_create.go [hunk 1]
```diff
@@ -481,7 +481,19 @@ func addOCIBindMounts(g *generate.Generator, mounts []*runtime.Mount, mountLabel
...
+		options := []string{"rbind"}
+		switch mount.GetPropagation() {
+		case runtime.MountPropagation_PROPAGATION_PRIVATE:
+			options = append(options, "rprivate")
+		case runtime.MountPropagation_PROPAGATION_BIDIRECTIONAL:
+			options = append(options, "rshared")
+		case runtime.MountPropagation_PROPAGATION_HOST_TO_CONTAINER:
+			options = append(options, "rslave")
+		default:
+			glog.Warningf("unknown propagation mode for hostPath %q", mount.HostPath)
```
Comment (Random-Liu) [inline pkg/server/container_create.go]: nit: s/unknown/Unknown

Log should start with capitalized letter. :)

**Comment by Random-Liu** [comment]
Based on the comment https://github.com/kubernetes/kubernetes/pull/46444#issuecomment-304054451, there will be a conformance test for mount propagation. Please make sure we enable that conformance test in our node e2e after they add it. :)

**Comment by Random-Liu** [comment]
@miaoyq Could you add a CRI validation test for this? https://github.com/kubernetes-incubator/cri-tools/issues/131

Just mount in container and host, and check whether the visibility of the mounts in different mode. :)

File: pkg/server/container_create.go (hunk 1 above)
Comment (miaoyq) [reply pkg/server/container_create.go]: @Random-Liu  Thank for review, Done.

**Comment by miaoyq** [comment]
>@miaoyq Could you add a CRI validation test for this? kubernetes-incubator/cri-tools#131

Just mount in container and host, and check whether the visibility of the mounts in different mode. :)

I will work for this. : )

**Comment by Random-Liu** [comment]
LGTM, will merge after test passes.

**Comment by Random-Liu** [comment]
The test has passed, I don't know why the label is still yellow. Merge anyway.

**Pull request merged**